sql/process_ccw_table.sql

```diff
@@ -0,0 +1,264 @@
+CREATE OR REPLACE PROCEDURE load_from_ccw()
```

Comment:
I think this fails to account for records that should be deleted? For instance, if a bene-year record _is_ in the DB but _isn't_ in this one-time export, shouldn't we remove it?

Reply:
hmmm...tell me if this use case is correct:
- we have a BeneficiaryMonthly rcd for 2020-03-01.
- in our 2020 CCW data feed we don't have a rcd for 2020-03-1 for that same beneficiary.
- ideally we'd like to nuke the 2020-03-01 rcd from BeneficiaryMonthly table.

I'm thinking the best way to handle that would be to do a post-process exercise (query) where we select all bene_id, yearMonth from BeneficiaryMonthly MINUS all bene_id, yearMonth audit table (should provide a list of rcds we'd want to nuke). So wrap the above in a delete from where bene_id, yearMonth in(...
or something like that.

If we are going to do that, then I guess I'd also like to add a delete trigger; always good to see where things!

Thoughts?

Reply:
> tell me if this use case is correct

Yup! I can't really work out how _likely_ it is, but better safe than sorry, I think.

> I'm thinking the best way to handle that would be to do a post-process exercise (query) where we select all bene_id, yearMonth from BeneficiaryMonthly MINUS all bene_id, yearMonth audit table (should provide a list of rcds we'd want to nuke). So wrap the above in a delete from where bene_id, yearMonth in(...
or something like that.

Agreed.

> If we are going to do that, then I guess I'd also like to add a delete trigger; always good to see where things go!

Also agreed!